fastly/resource_fastly_tls_subscription.go

```diff
@@ -73,6 +73,31 @@ func resourceFastlyTLSSubscription() *schema.Resource {
 				Description: "The details required to configure DNS to respond to ACME DNS challenge in order to verify domain ownership.",
 				Computed:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
+				Deprecated:  "Use 'managed_http_challenges' attribute instead",
```

Comment:
I think the message should be "Use 'managed_**dns**_challenges' attribute instead" instead shouldn't it?

Reply:
fixed